README.md

```diff
@@ -164,6 +164,7 @@ have a suggestion for another dataset to include in PyKEEN, please let us know
 | DistMult Literal (Gated)       | [`pykeen.models.DistMultLiteralGated`](https://pykeen.readthedocs.io/en/latest/api/pykeen.models.DistMultLiteralGated.html) | [Kristiadi *et al.*, 2018](https://arxiv.org/abs/1802.00934)                                                            |
 | ER-MLP                         | [`pykeen.models.ERMLP`](https://pykeen.readthedocs.io/en/latest/api/pykeen.models.ERMLP.html)                               | [Dong *et al.*, 2014](https://dl.acm.org/citation.cfm?id=2623623)                                                       |
 | ER-MLP (E)                     | [`pykeen.models.ERMLPE`](https://pykeen.readthedocs.io/en/latest/api/pykeen.models.ERMLPE.html)                             | [Sharifzadeh *et al.*, 2019](https://github.com/pykeen/pykeen)                                                          |
+| Fixed Model                    | [`pykeen.models.FixedModel`](https://pykeen.readthedocs.io/en/latest/api/pykeen.models.FixedModel.html)                     | [Berrendorf *et al.*, 2021](https://github.com/pykeen/pykeen/pull/691)                                                  |
```

Comment:
do we want to have the mock model be part of the model list?

Reply:
Yes!